src/constants/surveys.js

```diff
@@ -36,3 +35,8 @@ export const NEW_SURVEY = Object.freeze({
     },
   },
 });
+
+// Used to indicate the status of a survey
```

Comment:
Nit: Can you add a period at the end of the comment to keep it consistent with other comments?